x-pack/packages/kbn-entities-schema/src/rest_spec/common.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+export const booleanStringSchema = z
```

Comment:
The `@kbn/zod` package has a `BooleanFromString` helper, that given a boolean string ("true", "false") gives a boolean primitive, does this do the same thing?

See https://github.com/elastic/kibana/pull/191048